model/SimpleLicensing/Properties/customIdToLicense.md

```diff
@@ -0,0 +1,29 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# customIdToLicense
+
+## Summary
+
+Maps a LicenseRef or AdditionRef string for a Custom License or a Custom
+License Addition to a CustomLicense, a CustomLicenseAddition, or a
+SimpleLicensingText
```

Comment:
I'm not a native English speaker and I find this long paragraph with three "or"s and "for" and "to" quite confusing.

Does there a way to make this clearer? Also for the benefit of our translators as well.

And does there a difference between "Custom License" (with a space between two words) and "CustomLicense" (without space) ?

Reply:
Does this a correct semantic?

"Maps (a LicenseRef or AdditionRef string) for (a Custom License or a Custom
License Addition) to (a CustomLicense, a CustomLicenseAddition, or a
SimpleLicensingText)."

Reply:
```suggestion
Maps a LicenseRef or AdditionRef string for a custom license or a custom
license addition to a CustomLicense, a CustomLicenseAddition, or a
SimpleLicensingText.
```

If "Custom License" and "Custom License Addition" are not proper nouns or specific terms that casing is sensitive, can we use "custom license" and "custom license addition" instead? This will make them more different from the class names.